tests/sip/test_client.py

```diff
@@ -240,3 +243,45 @@ def test_patron_password_is_optional(self):
         assert with_password.endswith(
             'AApatron_identifier|AC|ADpatron_password'
         )
+
+    def test_parse_patron_status(self):
+        m = MockSIPClient.parse_patron_status
+        assert_raises(ValueError, m, None)
+        assert_raises(ValueError, m, "")
+        assert_raises(ValueError, m, " " * 20)
+        parsed = m("Y Y Y Y Y Y Y ")
+        for yes in [
+                'charge privileges denied',
+                #'renewal privileges denied',
+                'recall privileges denied',
+                #'hold privileges denied',
+                'card reported lost',
+                #'too many items charged',
+                'too many items overdue',
+                #'too many renewals',
+                'too many claims of items returned',
+                #'too many items lost',
+                'excessive outstanding fines',
+                #'excessive outstanding fees',
+                'recall overdue',
+                #'too many items billed',
```

Comment:
Yes, it's a way to visualize the values I expect to be set to true versus the entire list of values.